tests/unit/test_jenkins.py

```diff
@@ -1506,3 +1506,65 @@ def test_remove_unlisted_plugins(  # pylint: disable=too-many-arguments
         )
     else:
         mock_client.delete_plugins.assert_not_called()
+
+
+def test__invalidate_sessions(
```

Comment:
I can't see the point of adding this test

Reply:
This was solely done to increase the coverage tbh.